### ISSUE TITLE
DatabaseIDModel matching query does not exist on Device Info tab

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#2801]
#### beta3 on Windows 7 VM

Errors on Device Info just after installing during the bug bash; logged in as Admin, so not sure if it was supposed to be fixed by #2737 
 
![win7 start running - oracle vm virtualbox_101](https://user-images.githubusercontent.com/1457929/33511580-6c252b74-d71d-11e7-8409-cc98bc9ea90d.png)

#### beta5 on Ubuntu 16.10 VM

Fresh PEX run, login as Admin and go straight to Device info:

![fullscreen_02_12_17_19_32](https://user-images.githubusercontent.com/1457929/33518611-eedf7572-d797-11e7-9178-0e3dd3dad48c.png)

![fullscreen_02_12_17_19_33](https://user-images.githubusercontent.com/1457929/33518698-c891c350-d799-11e7-952a-7f9c157e9cfa.png)

However, upon restarting the server/VM, Device info displays correctly:

![fullscreen_02_12_17_18_50](https://user-images.githubusercontent.com/1457929/33518434-31b4cf4e-d795-11e7-86c4-f7d69a285656.png)

### Expected behavior

Seeing the Device Info page with no errors upon installation. 

### Errors and logs

Relevant errors and tracebacks from:

* the command line
```
404 static files copied to '/home/osboxes/.kolibri/static'.
Operations to perform:
  Apply all migrations: sessions, admin, kolibriauth, morango, auth, content, contenttypes, exams, device, logger
Running migrations:
  No migrations to apply.
  Your models have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
INFO     Running 'kolibri start' in foreground...
INFO     Kolibri running on:

	http://127.0.0.1:8080/
	http://10.0.2.15:8080/
	http://192.168.1.46:8080/

Installed 2 object(s) from 1 fixture(s)
INFO     Started new MESSAGEPROCESSOR thread ID#oranges-seventeen-mobile-high
INFO     Started new SCHEDULER thread ID#spaghetti-mars-tennessee-foxtrot
INFO     Started new WORKERMESSAGEHANDLER thread ID#march-quiet-berlin-uranus
ERROR    Internal Server Error: /api/deviceinfo/
Traceback (most recent call last):
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/core/device/api.py", line 74, in get
    instance_model = InstanceIDModel.get_or_create_current_instance()[0]
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/morango/models.py", line 89, in get_or_create_current_instance
    "database": DatabaseIDModel.objects.get(current=True),
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/db/models/manager.py", line 122, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/db/models/query.py", line 387, in get
    self.model._meta.object_name
DoesNotExist: DatabaseIDModel matching query does not exist.
ERROR    Internal Server Error: /api/deviceinfo/
Traceback (most recent call last):
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 149, in get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 147, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 466, in dispatch
    response = self.handle_exception(exc)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 463, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/core/device/api.py", line 74, in get
    instance_model = InstanceIDModel.get_or_create_current_instance()[0]
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/morango/models.py", line 89, in get_or_create_current_instance
    "database": DatabaseIDModel.objects.get(current=True),
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/db/models/manager.py", line 122, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/osboxes/.pex/install/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl.f4b3e2ea8dc3fd5376c09a92f321cc5f944ced42/kolibri-0.7.0.dev020171201172142_git-py2.py3-none-any.whl/kolibri/dist/django/db/models/query.py", line 387, in get
    self.model._meta.object_name
DoesNotExist: DatabaseIDModel matching query does not exist.
```
[kolibri.log](https://github.com/learningequality/kolibri/files/1524061/kolibri.log)

### Steps to reproduce

Run the PEX and go to Device info after the setup wizard

### Context

* Kolibri version: beta5 - kolibri-0.7.0.dev020171201172142-git.pex
* Operating system: Ubuntu 16.10
* Browser: Chrome & FF

cc @indirectlylit @christianmemije @jonboiser

**Comment by indirectlylit** [comment]
yes, this should be fixed in the latest beta. Please re-open if you see it again

**Comment by radinamatic** [comment]
@indirectlylit Just making sure you noticed: the first error (Windows VM) was on beta3 during bug bash, but the second one was on Ubuntu with [beta5, which is the latest](https://github.com/learningequality/kolibri/releases/tag/v0.7.0-beta5)... right?

**Comment by indirectlylit** [comment]
thanks @radinamatic I did not notice that! #2802 is the same issue, so we'll address it there

**Comment by radinamatic** [comment]
Just as long as it's gone! 💥

**Comment by ralphiee22** [comment]
Will be resolved in morango-0.2.0.
